web_dashboard.py

```
@@ -0,0 +1,137 @@
+    except Exception as e:
+        print(f"API settings error: {e}")
+    return jsonify({})
+
+
+@app.route("/api/settings", methods=["POST"])
+def api_update_settings():
+    try:
+        data = request.get_json(force=True)
+        if _update_settings_fn:
+            return jsonify(_update_settings_fn(data))
+    except Exception as e:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)